motion-1/Overview.bs

```diff
@@ -163,19 +235,235 @@ Media: visual
+			offset-distance: 100%;
+		}
+	&lt;/style>
+</pre>
+<div class=figure> 
+	<img alt="An image of three elements positioned to polar coordinates" src="images/simple_offset_position.png" style="width: 300px;"/>
+	<figcaption>An example of positioning element in polar coordinates</figcaption>
+</div>
+</div>
 
-<dl dfn-for="motion-offset" dfn-type="value">
-  <dt><<length-percentage>></dt>
-  <dd>The distance from the <a>initial position</a> of the shape or path to the position of the element. Percentages are relative to the total path length including all sub-paths.</dd>
+<h3 id="offset-position-property">Define the initial position of the path: The 'offset-position' property</h3>
+<pre class='propdef'>
+Name: offset-position
+Applies to: all elements
+Value: auto | <<position>>
+Initial: auto
+Media: visual
+Inherited: no
+Percentages: Refer to the size of containing block
+Animatable: as <a href="https://www.w3.org/TR/css3-transitions/#animtype-lpcalc">position</a>
+</pre>
+Specifies the initial position of the path.
+
+Values are defined as follows:
+<dl dfn-for="offset-position" dfn-type="value">
+<dt><var>'auto'</var></dt>
+<dd>Computes to the current position of the element.</dd>
+
+<dt><<position>></dt>
+<dd>The <<position>> represents the horizontal position and the vertical position 
+	as specified in CSS Backgrounds [[CSS3-BACKGROUND]].
+
+	Values of <<position>> are defined like below:
+	<dl>
+		<dt><<percentage>></dt>
+		<dd>A percentage for the horizontal offset is relative to the width of 
+		the containing block.
+		A percentage for the vertical offset is relative to the height of 
+		the containing block.
+		</dd>
+
+		<dt><<length>></dt>
+		<dd>A length value gives the fixed length as an offset.
+		The offset is from the top left corner of the containing block.
+		</dd>
+
+		<dt><dfn>top</dfn></dt>
+		<dd>Computes to 0% for the vertical position.</dd>
+
+		<dt><dfn>right</dfn></dt>
+		<dd>Computes to 100% for the horizontal position.</dd>
+
+		<dt><dfn>bottom</dfn></dt>
+		<dd>Computes to 100% for the vertical position.</dd>
+
+		<dt><dfn>left</dfn></dt>
+		<dd>Computes to 0% for the horizontal position.</dd>
+
+		<dt><dfn>center</dfn></dt>
+		<dd>Computes to 50% (left 50%) for the horizontal position 
+		if the horizontal position is not otherwise specified, 
+		or 50% (top 50%) for the vertical position if it is.</dd>
+	</dl>
+</dd>
+</dl>
+
+<h3 id="offset-anchor-property">Define an anchor point: The 'offset-anchor' property</h3>
+<pre class='propdef'>
+Name: offset-anchor
+Applies to: All elements
+Value: <<position>>
+Initial: 'auto'
+Media: visual
+Inherited: no
+Percentages: Relative to the width and the height of an element
+Animatable: as <<position>>
+</pre>
+
+Defines an anchor point of the element positioned along the path. 
+The anchor specifies a position which is a representative point of the element 
+and could be set as any point within the content area of the element. 
+The anchor point is placed along the path.
```

Comment:
"The anchor specifies the point of the element which is to be considered as the point that is moved along the path"?